src/app/ui/editor/tool_loop_impl.cpp

```diff
@@ -247,6 +247,16 @@ class ToolLoopBase : public tools::ToolLoop {
     }
     return false;
   }
+
+  bool isPixelConnectivityFourConnected() override {
```

Comment:
I think this could be simplified to something like:
```c++
  return m_toolPref.floodfill.pixelConnectivity() == app::gen::PixelConnectivity::FOUR_CONNECTED;
```